scripts/ci-conformance.sh

```diff
@@ -36,7 +36,10 @@ cleanup() {
 }
 trap cleanup EXIT
 
-#Install requests module explicitly for HTTP calls
+# Set correct pip3.7 location for python3.
+ln -s /usr/local/bin/pip3.7 /usr/bin/pip3.7
```

Comment:
This would blow up if it already exists though? 🤔  Isn't that the case for the current image?

Reply:
`/usr/bin/pip3.7` does not exist in the current image, that's the point.

WDYT about

```suggestion
if ! test -f /usr/bin/pip3.7 && test -f /usr/local/bin/pip3.7; then
	ln -s /usr/local/bin/pip3.7 /usr/bin/pip3.7
fi
```

Reply:
Or` we could easily install it via `apt install python3-requests -y`

Reply:
I'm cool with the `if`! :)